examples/stable-diffusion/training/train_text_to_image_sdxl.py

```diff
@@ -915,16 +834,21 @@ def preprocess_train(examples):
         for image in images:
             original_sizes.append((image.height, image.width))
             image = train_resize(image)
+            if args.crop_resolution < args.resolution:
```

Comment:
Can you explain a bit more why `args.crop_resolution` is introduced?

Reply:
since we have resize https://github.com/huggingface/optimum-habana/blob/yyoon_stdxl/examples/stable-diffusion/training/train_text_to_image_sdxl.py#L836. only if the crop_resolution is less than the resized image, then we crop. for performance improvement